packages/@stimulus/test/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@stimulus/test",
+  "version": "0.9.0",
+  "description": "Stimulus JavaScript framework: Test helpers",
+  "repository": "https://github.com/stimulusjs/stimulus",
+  "author": "Basecamp, LLC",
+  "contributors": [
+    "Javan Makhmali <javan@javan.us>",
+    "Sam Stephenson <sstephenson@gmail.com>"
+  ],
+  "main": "dist/module/index.js",
+  "types": "dist/types/index.d.ts",
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  }
+}
```

Comment:
This package should probably depend on `qunit` and `@stimulus/core`

Reply:
I wasn’t sure how to specify the QUnit dependency, since we don’t import/require it anywhere in this package. 

Adding `@stimulus/core` as a dependency would be cyclical - is that right?